operators/emergent_source/README.md

```diff
@@ -0,0 +1,93 @@
+# Emergent Source Operator
+
+The `emergent_source` operator provides support for Emergent Vision Technologies cameras as video sources. This operator enables high-performance video streaming through Mellanox ConnectX SmartNIC using the Rivermax SDK.
+
+## Overview
+
+The EmergentSourceOp is designed to capture video streams from Emergent Vision Technologies cameras with high frame rates and resolution support. It leverages RDMA (Remote Direct Memory Access) capabilities for efficient data transfer and supports various camera parameters for optimal performance.
```

Comment:
It would be good to reference [Holoscan EVT setup docs](https://docs.nvidia.com/holoscan/sdk-user-guide/emergent_setup.html) in this README and/or in the high speed endoscopy app README